components/Map/Map.js

```diff
@@ -85,7 +85,7 @@ let events = [
     return (
     <MapView
     showsUserLocation={true}
-    style={{height: 350}}
+    style={styles.map}
```

Comment:
Glad we moved the styling from inline to the bottom of the component. This is def better for readability.